src/js/app.js

```diff
@@ -1,26 +1,31 @@
 import { buildData, filterObjectsOnNodeClick, filterNodes } from './data-builder';
 import renderNetwork from './network-vis';
 import renderHierarchy from './hierarchy-vis';
+import '../../node_modules/jquery-ui-dist/jquery-ui';
+import '../../node_modules/jquery-ui-dist/jquery-ui.min.css';
 
-let data = {};
+// Needed for node search functionality
+let allNodeNames = [];
 
+let data = {};
 const init = (rawData) => {
```

Comment:
Please comment your function definitions

Reply:
Self explanatory. function definition comment is noise than helpful in this case